fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraFixity.java

```diff
@@ -99,6 +117,13 @@ public RdfStream getDatastreamFixity() {
         }
 
         LOGGER.info("Get fixity for '{}'", externalPath);
+
+        final RdfStream rs = ((FedoraBinary)resource()).getFixity(translator())
```

Comment:
If we ever have any intention of carrying through the API partitioning as designed in the fall, we should place this in such a position that it will be easy to extract into a separate module later, along with `FedoraFixity` and associated gear.